docs/spec/spec.md

```diff
@@ -0,0 +1,310 @@
+
+---
+
+## kind: Subscription
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a linkage between a Subscribable and a Targetable and/or Sinkable._
+
+### Object Schema
+
+#### Spec
+
+| Field              | Type         | Description                                                                  | Limitations        |
+| ------------------ | ------------ | ---------------------------------------------------------------------------- | ------------------ |
+| from\*             | ObjectRef    | The originating Channel for the link.                                        | Must be a Channel. |
+| call<sup>1</sup>   | EndpointSpec | Optional processing on the event. The result of call will be sent to result. |                    |
+| result<sup>1</sup> | ObjectRef    | The continuation Channel for the link.                                       | Must be a Channel. |
+
+\*: Required
+
+1: At Least One(call, result)
+
+#### Metadata
+
+##### Owner References
+
+- If a resource controller created this Subscription: Owned by the originating
+  resource.
+
+##### Conditions
+
+- **Ready.**
+- **FromReady.**
+- **CallActive.** True if the call is sinking events without error.
+- **Resolved.**
+
+#### Events
+
+- PublisherAcknowledged
+- ActionFailed
+
+### Life Cycle
+
+| Action | Reactions                                                        | Limitations |
+| ------ | ---------------------------------------------------------------- | ----------- |
+| Create | The publisher referenced needs to be watching for Subscriptions. |             |
+| Update |                                                                  |             |
+| Delete |                                                                  |             |
+
+---
+
+## kind: EventType
+
+NOTE: EventType is out of scope for 0.1 release. This is future documentation.
+
+### group: eventing.knative.dev/v1alpha1
+
+_Describes a particular schema of Events which may be produced by one or more
+source systems._
+
+### Object Schema
+
+#### Spec
+
+| Field      | Type   | Description                                         | Limitations                    |
+| ---------- | ------ | --------------------------------------------------- | ------------------------------ |
+| jsonSchema | String | The Json Schema that represents an event in flight. | Only for JSON transport types. |
+
+#### Metadata
+
+##### Owner References
+
+EventType is owned by _Provisioners_. Each _Provisioner_ creates a
+non-controlling OwnerReference on the EventType resources it knows about.
+
+#### Status
+
+| Field          | Type    | Description                         | Limitations |
+| -------------- | ------- | ----------------------------------- | ----------- |
+| referenceCount | Integer | Number of Owners for this EventType |             |
+
+##### Conditions
+
+- **Pinned**. True when the EventType cannot be deleted.
```

Comment:
Recommend that we don't have Conditions on EventTypes. We may not want to have Spec/Status, either, since EventType is just "raw data". (See also [ConfigMap](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#configmap-v1-core) for an example of such a resource.)

Reply:
Yes we need to revisit it in 0.1+